h/sql_tasks/tasks/report/fast_recreate/00_permissions/01_grant_schema.jinja2.sql

```diff
@@ -1,4 +1,17 @@
+-- The public schema already exists but
+-- we need to grant usage to the user will map via fdw.
+{% for fdw_user in fdw_users %}
+GRANT USAGE ON SCHEMA public TO "{{fdw_user}}";
+GRANT SELECT ON  public.user_group TO "{{fdw_user}}";
+GRANT SELECT ON  public."user" TO "{{fdw_user}}";
+GRANT SELECT ON  public."group" TO "{{fdw_user}}";
+{% endfor %}
+
 {% for fdw_user in fdw_users %}
 GRANT USAGE ON SCHEMA report TO "{{fdw_user}}";
 GRANT SELECT ON ALL TABLES IN SCHEMA report TO "{{fdw_user}}";
 {% endfor %}
+
+
+-- Empty query in case fdw_users is empty.
+select null;
```

Comment:
We should fix this in the task runner, if there no queries on the file, don't run it.

Reply:
It's hard to spot there aren't any if there are any comments unfortunately. It would be neat if we could detect it's empty.

This could be as easy as stripping comments and then checking if it's blank. Woe betide you if you have `--` somewhere in the query though.

Reply:
Also, should we add this to `tox.ini` so there is a user?